### ISSUE TITLE
Error handling - [Errno 111] Connect call failed ('10.10.10.2', 22)

### ISSUE THREAD

**Issue by ajwillo** [selfuryon/netdev#12]
How do we handle errors when a device cannot be connected to? currently I et

  File "/usr/local/lib/python3.6/asyncio/selector_events.py", line 480, in _sock_connect_cb
    raise OSError(err, 'Connect call failed %s' % (address,))
ConnectionRefusedError: [Errno 111] Connect call failed ('10.10.10.2', 22)

I would just like to print an exception i.e. couldn't connect to X. at the moment it aborts my script

Thanks

**Comment by selfuryon** [comment]
Yes, it's right solution about using exception handling! One note about it - you may want to use `asyncio.gather` instead of `asyncio.wait` for awaiting coroutines due to `wait` doesn't catch the exception by default - you can read about it [here](https://medium.com/@yeraydiazdiaz/asyncio-coroutine-patterns-errors-and-cancellation-3bb422e961ff)
